src/styles/calendar.scss

```
@@ -0,0 +1,51 @@
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  width: $day-rows * $cell-size + $padding-calendar * 2 + $gap * 7;
+  gap: $gap;
+  padding: $padding-calendar;
+
+  &__day {
+    box-sizing: border-box;
+    border: 1px solid black;
+    background-color: $background-color;
+    height: $cell-size;
+    width: $cell-size;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    transition: all;
+    transition-duration: .5s;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```